File: app/src/main/java/com/moon/beautygirlkotlin/utils/Logger.java
package com.moon.beautygirlkotlin.utils;

import android.util.Log;

/**
 * Created by sprit_moon on 15/9/9.
 */
public class Logger {


    private static final String TAG = "moon";
    
    public static boolean DEBUG = false;

    public static void i(String value) {
        if (DEBUG) {
            Log.i(TAG, value);

        }
    }

    public static void i(String tag,String value) {
        if (DEBUG) {
            Log.i(tag, value);
        }
    }


    public static void e(String tag, String value) {
        if (DEBUG) {
            Log.e(tag, value);
        }
    }

    public static void d(String value) {
        if (DEBUG) {
            Log.d(TAG, value);
        }

    }

    public static void v(String value) {
        if (DEBUG) {

            Log.v(TAG, value);
        }

    }

    public static void e(String value) {
        if (DEBUG) {

            Log.e(TAG, value);
        }
    }
}

File: app/src/main/java/com/moon/beautygirlkotlin/utils/Logger.kt
package com.moon.beautygirlkotlin.utils

import android.util.Log

/**
 * author: moon
 * created on: 18/5/9 下午6:32
 * description:
 */
object Logger {


    private val TAG = "moon"

    var DEBUG = true

    fun i(value: String) {
        if (DEBUG) {
            Log.i(TAG, value)

        }
    }

    fun i(tag: String, value: String) {
        if (DEBUG) {
            Log.i(tag, value)
        }
    }


    fun e(tag: String, value: String) {
        if (DEBUG) {
            Log.e(tag, value)
        }
    }

    fun d(value: String) {
        if (DEBUG) {
            Log.d(TAG, value)
        }

    }

    fun v(value: String) {
        if (DEBUG) {

            Log.v(TAG, value)
        }

    }

    fun e(value: String) {
        if (DEBUG) {

            Log.e(TAG, value)
        }
    }

}